sdk/keyvault/keyvault-keys/package.json

```diff
@@ -51,14 +52,16 @@
     "execute:ts-samples": "node ../../../common/scripts/run-samples.js samples/typescript/dist/samples/typescript/src/",
     "execute:samples": "npm run build:samples && npm run execute:js-samples && npm run execute:ts-samples",
     "extract-api": "tsc -p . && api-extractor run --local",
-    "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\" \"keyvault-common/**/*.ts\"",
     "integration-test:browser": "karma start --single-run",
     "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- --timeout 5000000 --full-trace dist-esm/test/*.test.js",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint package.json tsconfig.json src test samples --ext .ts --fix --fix-type [problem,suggestion]",
     "lint": "eslint package.json tsconfig.json src test samples --ext .ts -f html -o keyvault-keys-lintReport.html || exit 0",
     "lint:terminal": "eslint package.json tsconfig.json src test samples --ext .ts",
+    "prepack": "bash ../prepack.sh",
```

Comment:
I mean, this won't work in command prompt on windows, yeah?

Reply:
Nope! This is only a draft. If we end up picking this approach, I can write something on another language (let's say, JavaScript 😋 )